prime-dotnet-webapi/Services/EmailService.cs

```diff
@@ -7,33 +10,100 @@ namespace Prime.Services
 {
     public class EmailService : BaseService, IEmailService
     {
-        public EmailService(
-            ApiDbContext context, IHttpContextAccessor httpContext)
+        private const string PRIME_EMAIL = "prime@gov.bc.ca";
```

Comment:
I'm 50-50 if this should be an env variable

Reply:
Would you want it to be different between dev/test/prod?

Reply:
No, and it's why I'm 50-50. But this email is highly likely to change.